addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10313,7 +10313,11 @@ msgctxt "#20244"
 msgid "Android Apps"
 msgstr ""
 
-#empty strings from id 20245 to 20249
+msgctxt "#20245"
```

Comment:
In this instance, settings are just like Android apps. They are shortcuts to the one and only droid settings app, each shortcut opening the proper page/tab.
AFAIK, there is just no way for a normal app like us to access the underlying values.

An alternative would be to mix them with the normal apps vfs (as it's basically the same thing), but would be more cumbersome, imo.